src/mantine-core/src/Radio/Radio.styles.ts

```diff
@@ -118,6 +118,8 @@ export default createStyles(
 
       labelWrapper: {
         ...theme.fn.fontStyles(),
+        display: 'flex',
+        alignItems: 'flex-start',
```

Comment:
In my opinion, it would be "center", not flex-start, it's weird I guess.

Reply:
Yeah, makes sense. I just changed it to how it was before these properties were removed